### PR TITLE
Test/surveys

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: src/state/surveys/sagas/index.js [hunk 1]
```diff
@@ -10,8 +13,9 @@ import surveysActions from '../actionCreators';
 import surveysActionTypes from '../actionTypes';
 import surveysApi from '../api';
 
-function* handleFetchSurvey({payload: {surveyId}} = {}) {
```
Comment (SteRiccio) [inline src/state/surveys/sagas/index.js:13]: the survey could be fetched by uuid

File: .gitignore [hunk 2]
```diff
@@ -68,6 +68,7 @@ buck-out/
 
 # Test
 coverage/*
+test_remort.xml
```
Comment (SteRiccio) [inline .gitignore]: this should be test-report.xml!

**Review by SteRiccio — CHANGES_REQUESTED** [review]

File: src/state/surveys/sagas/index.js (hunk 1 above)
Comment (ramirobg94) [reply src/state/surveys/sagas/index.js:13]: next PR

Comment (ramirobg94) [reply src/state/surveys/sagas/index.js:13]: Lets do in other PR

File: .gitignore (hunk 2 above)
Comment (ramirobg94) [inline .gitignore]:
```suggestion
test_report.xml
```

**Review by SteRiccio — APPROVED** [review]

**Pull request merged**